babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -663,6 +663,39 @@ SqlNode InlineModOperator() :
     }
 }
 
+SqlNode CurrentDateTimeFunctions() :
```

Comment:
I think adding this caused the build issue, anyone have any idea of how it could be fixed?